builtin/credential/cert/path_certs.go

```diff
@@ -228,6 +234,15 @@ func (b *backend) pathCertWrite(ctx context.Context, req *logical.Request, d *fr
 		}
 	}
 
+	var parsedCIDRs []*sockaddr.SockAddrMarshaler
+	for _, v := range d.Get("bound_cidrs").([]string) {
+		parsedCIDR, err := sockaddr.NewSockAddr(v)
+		if err != nil {
+			return nil, err
```

Comment:
Can we make this a `logical.ErrorResponse` to indicate the user that this is an input error as opposed to a 500?

Reply:
:heavy_check_mark: